### ISSUE TITLE
Mothership Cloaking Field Behaviour displays as "Unknown" on Unit Info Panel 

### ISSUE THREAD

**Issue by Joshua-Leibold** [Ahli/sc2xml#116]
The mothership received a new ability that cloaks units for a duration after activated in Patch 5.0.12 
While the mothership is cloaking units surrounding it, it has a behaviour. 
This behaviour tooltip title displays incorrectly as "Unknown [CloakField]" instead of just "Cloak Field"

https://i.gyazo.com/115dfde80b96097284fd8656b01951e6.mp4

I wish I knew what was causing it

**Comment by Ahli** [comment]
A fix would be to edit the behavior to use the same name as the other behavior that is added to the cloaked units.
```xml
    <CBehaviorBuff id="CloakField">
        <Name value="Behavior/Name/CloakFieldEffect"/>
    </CBehaviorBuff>
```

**Comment by Joshua-Leibold** [comment]
Successfully patched live with 5.0.13
